docs/index.asciidoc

```diff
@@ -234,9 +235,21 @@ Verify the identity of the other end of the SSL connection against the CA.
 use label parsing for severity and facility levels
 use priority field if set to false
 
+[id="plugins-{type}s-{plugin}-structured_data"]
+===== `structured_data`
 
+  * Value type is <<string,string>>
+  * There is no default value for this setting.
+
+RFC5424 structured data is a string of one or more structured data elements, including brackets.
+The elements need to be formatted according to link:https://datatracker.ietf.org/doc/html/rfc5424#section-6.3[RFC5424 section 6.3], for example:
+
+[source,ruby]
```

Comment:
This is not Ruby code, so no need for that highlighting:
```suggestion
["source",subs="attributes"]
```

Reply:
Just to double-check: `"source"` should be in quotes? 

I'm not that familiar with asciidoc and I did not see it used that way before. [The doc](https://docs.asciidoctor.org/asciidoc/latest/attributes/element-attributes/) did not seem to explicitly mention quoted element attributes.

Reply:
If I recall correctly is indifferent, but looking the `logstash-input-beats` doc, the double quotes is used.